templates/zerver/emails/missed_message.source.html

```diff
@@ -45,9 +45,11 @@
 
 
     <p>
-        <a href="{{ realm_uri }}">Click here to log in to Zulip and view your new messages.</a>
```

Comment:
Here I have just changed something like

`[click here to log in to Zulip and view your new messages]`

to  something like `Reply to this email directly, or [view it in Zulip]`

and if `reply_to_zulip` is **false** then
 `Reply to this email directly, or [view it in Zulip]`